src/index.html

```diff
@@ -11,8 +11,9 @@
 <body>
   <div class="container">
     <img class="logo" src="/src/assets/logo.svg" alt="" />
-    <h1>Semantic HTML tags</h1>
-    <h2>How many do you know?</h2>
+    <h1>Tagged! The HTML Quiz</h1>
+    <h2>The ultimate challenge for web developers. Tag, you're it!</h2>
+    <h3>User Guide</h3>
```

Comment:
I would not add this to this PR. For the user guide, I am thinking more of an `About` link in the upper right corner - this would need more discussion and unnecessary rearrange when we decide on this feature.